netbeans.apache.org/src/content/download/nb16/index.adoc

```diff
@@ -75,36 +76,37 @@ link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/Apache-Net
 * link:{url-download}netbeans-installers/{netbeans-version}/apache-netbeans-{netbeans-version}-0.noarch.rpm[apache-netbeans-{netbeans-version}-0.noarch.rpm] (link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/apache-netbeans-{netbeans-version}-0.noarch.rpm.sha512[SHA-512],
 link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/apache-netbeans-{netbeans-version}-0.noarch.rpm.asc[PGP ASC])
+* link:https://snapcraft.io/netbeans[Linux snap package]
+
+*Source:*
 
-- Source: link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip[netbeans-{netbeans-version}-source.zip] (link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.sha512[SHA-512],
+* link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip[netbeans-{netbeans-version}-source.zip] (link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.sha512[SHA-512],
 link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.asc[PGP ASC])
 
-////
-NOTE: Using https below is highly recommended.
-////
 Officially, it is important that you link:https://www.apache.org/dyn/closer.cgi#verify[verify the integrity]
 of the downloaded files using the PGP signatures (.asc file) or a hash (.sha512 files).
 The PGP keys used to sign this release are available link:https://downloads.apache.org/netbeans/KEYS[here].
 
-Apache NetBeans can also be installed as a self-contained link:https://snapcraft.io/netbeans[snap package] on Linux.
-
 == Community Installers
 
-IMPORTANT: Individual NetBeans committers may provide additional binary packages as a convenience.
-While built using the Apache NetBeans release, they are not releases of the Apache Software
-Foundation. They may include other contents (eg. JDK) under additional license terms.
-
 - link:https://www.codelerity.com/netbeans/[Codelerity / Gj IT packages] - Windows, macOS and
 Linux (.deb / .rpm / .AppImage) built with
 link:https://github.com/apache/netbeans-nbpackage/[NBPackage]. Most
 include a local JDK runtime for the IDE to run on, for a self-contained out-of-the-box
-experience (other JDK's may be used for projects).
+experience.
+
+
+IMPORTANT: Individual NetBeans committers may provide additional binary packages as a convenience.
+While built using the Apache NetBeans release, they are not releases of the Apache Software
+Foundation. They may include other contents (eg. JDK) under additional license terms.
 
 == Deployment Platforms
 
 The Apache NetBeans {netbeans-version} binary releases require JDK 11+, and officially support running on JDK 11 and JDK 17.
 
-TIP: Gradle projects in Apache NetBeans 16 are not supported when running the IDE on JDK 19.
+TIP: The Runtime JDK NetBeans uses does not influence the JDK range projects can use.
```

Comment:
This is too small TIP. It would be better to have a `Known Issues` section. Also it would worth to mention that the team is planning  to release a patch, to fix those issues.

Reply:
> This is too small TIP

i don't understand what this means. This was moved from L101 since it applies to all artifacts and fits better to the deployment section IMO. You want me to remove this completely?

> Also it would worth to mention that the team is planning to release a patch, to fix those issues.

I actually commented the same on the last PR https://github.com/apache/netbeans-website/pull/631#discussion_r1042532670

So I added a 'currently' to the sentence but didn't change anything else. Once the update is out someone has to edit this anyway.

Reply:
> So I added a 'currently' to the sentence but didn't change anything else. Once the update is out someone has to edit this anyway.

Funny, I had 'currently' in there until the final draft. Probably better there in hindsight. I don't think the patch update needs to be specifically mentioned. And, yes, we'll have to edit later, with something that informs the user of the need to run updates after install.

Reply:
A Known Issue section might be good though.

Reply:
this page slowly mutates into a README

https://github.com/apache/netbeans-website/blob/afc6f1984febde6c5d860deddd3b0aeb97e0786f/netbeans.apache.org/src/content/download/nb16/index.adoc

tried to take all feedback into account the way I understood it.

Reply:
A good release shall come with a good release notes. The three most important thing in a release notes are:
 - Downloads 
 - Changelog
 - Known Issues

Our download page is a kind of release notes, we offloaded the changelog to github.